AnalysisStep/src/PhotonIDHelper.cc

```diff
@@ -17,11 +17,37 @@ bool PhotonIDHelper::isCutBasedID_Loose(int year, const pat::Photon& photon){
    
    bool isID = false;
    
-   // [FIXME] Removing photon ID for the moment since we do not use it in the analysis. Be careful if you want to use it, because the code will run locally but if you try to submit the jobs on Condor they will fail becuase dumpPython() replaces "-" with "_". I have contacted EGamma to provide a fix for this so maybe check if a new version of photon ID that does not have this problem exists before using it.
+   // [FIXME] Removing photon ID for the moment since we do not use it in the analysis. Be careful if you want to use it, because the code will run locally but if you try to submit the jobs on Condor they will fail becua replaces "-" with "_". I have contacted EGamma to provide a fix for this so maybe check if a new version of photon ID that does not have this problem exists before using it.
```

Comment:
Is this comment still relevant? It looks like the ID names were corrected so it can possibly be removed (also below)